src/Maybe.js

```diff
@@ -95,4 +97,12 @@ _Nothing.prototype.getOrElse = function(a) {
   return a;
 };
 
+_Just.prototype.toString = function() {
+  return 'Maybe.Just(' + R.toString(this.value) + ')';
+};
+
+_Nothing.prototype.toString = function() {
+  return 'Maybe.Nothing()';
```

Comment:
no strong feeling. i suppose the fact that there is only one `Nothing` in ramda-fantasy is an implementation detail